src/include/kernel/ext_manifest.h

```diff
@@ -73,6 +73,23 @@ enum ext_man_elem_type {
 	EXT_MAN_ELEM_FW_VERSION		= 0,
 	EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	EXT_MAN_ELEM_PROBE_INFO		= SOF_IPC_EXT_PROBE_INFO,
+	EXT_MAN_ELEM_CONFIG_DATA,
```

Comment:
I think it may be good habit to assign rigid value to enum value which is part of IPC, in this situation:
`EXT_MAN_ELEM_CONFIG_DATA = 4`
Then it will be not so easy to change enum value unintentionally in future.

Reply:
supre, its 4 now